app/src/main/java/com/example/foody/recipe_search/presentation_mvvm/SearchViewModel.kt

```diff
@@ -69,8 +69,8 @@ class SearchViewModel @Inject constructor(
                 if (recipeList.isEmpty()) RecipeListState.Empty
                 else RecipeListState.Success(recipeList = recipeList)
             } catch (e: Exception) {
-                Log.e("RecipeSearchViewModel", e.message.orEmpty(), e)
-                RecipeListState.Error("Unknown error from search.")
+                e.printStackTrace()
+                RecipeListState.Error("Unknown error from SearchVM.")
```

Comment:
This is a user facing message, thus should not be "technical".

Reply:
Yeah .... wrote it like that, and forgot.

Reply:
Corrected ...